src/FitterAlgoBase.cc

```diff
@@ -330,7 +334,7 @@ double FitterAlgoBase::findCrossing(CascadeMinimizer &minim, RooAbsReal &nll, Ro
     if (runtimedef::get("FITTER_NEW_CROSSING_ALGO")) {
         return findCrossingNew(minim, nll, r, level, rStart, rBound);
     }
-    ProfileLikelihood::MinimizerSentry minimizerConfig(minimizerAlgoForMinos_, minimizerToleranceForMinos_);
+    ProfileLikelihood::MinimizerSentry minimizerConfig(minimizerAlgoForMinos_, minimizerTolerance_);
```

Comment:
This shouldn't actually change the existing behaviour. Right now the minimizerToleranceForMinos_ was setting both Minuit's tolerance and the and the tolerance of the findCrossing algo. In principle these don't need to be tied together. In any case  the value of minimizerToleranceForMinos_ wasn't being used by minuit since we only call "improve" and not "minimize" here, and up to now calling "improve" doesn't change the existing RooMinimizer's tolerance setting. However the analytic barlow-beeston does cause the minimizer to be reset in "improve", picking up minimizerToleranceForMinos_ instead. Hence this change.